src/components/ImageView/index.native.js

```diff
@@ -76,13 +76,9 @@ class ImageView extends PureComponent {
 
             const aspectRatio = Math.min(containerHeight / imageHeight, containerWidth / imageWidth);
 
-            // Resize small images to fit the screen. Else resize the smaller dimension to avoid resize issue on Android - https://github.com/Expensify/App/pull/7660#issuecomment-1071622163
-            if (imageHeight < containerHeight && imageWidth < containerWidth) {
+            if ((imageHeight > containerHeight || imageWidth > containerWidth) && imageHeight > imageWidth) {
                 imageHeight *= aspectRatio;
-                imageWidth *= aspectRatio;
-            } else if (imageHeight > imageWidth) {
-                imageHeight *= aspectRatio;
-            } else {
+            } else if (imageHeight > containerHeight || imageWidth > containerWidth) {
                 imageWidth *= aspectRatio;
```

Comment:
Please clean up these conditions. Don't you think we should also set the resize mode to `center` like the web?

Reply:
if we change to `center` we cannot zoom small images on web and desktop. Can it stay as contain please? Because there is a feature to go to the clicked point in Desktop and Web versions.

Reply:
I think we need still `contain` because when we will use `center` zoom not working well. same as Web and Desktop version.
 
```js
            if (imageHeight > imageWidth) {
                imageHeight *= aspectRatio;
            } else {
                imageWidth *= aspectRatio;
            }
```
I change like this because still need this for bigger images.
otherwise look like this:

https://user-images.githubusercontent.com/6093207/165386321-1f128cef-c30c-47c3-9c60-fa5c4aba96f0.mp4